packages/client/src/components/TextandImage/index.js

```diff
@@ -12,11 +12,11 @@ img.circle{
 }
 `;
 
-const HorzIm = ({children, imgpath, shape}) => {
+const HorzIm = ({ children, imgpath, shape }) => {

```

Comment:
Nit: More descriptive name.

Reply:
Wups my linter just fixed a lot of errors within the page. I think that is Horizontal Image, but it isn't my component :grimacing: